api/src/main/resources/workbench-api.yaml

```diff
@@ -4807,6 +4807,17 @@ definitions:
         default: false
       kernelType:
         "$ref": "#/definitions/KernelTypeEnum"
+      includeMicroarrayData:
```

Comment:
It may make sense to start with an enum here for future extensibility. Like genomicsDataType: NONE, MICROARRAY (later: WHOME_GENOME)